src/DssSpell.sol

```diff
@@ -44,10 +73,79 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
         // Includes changes from the DssSpellCollateralOnboardingAction
         // onboardNewCollaterals();
 
-        // Deauthorize the existing lerp to prevent additional overwrites of hump.
-        DssExecLib.deauthorize(DssExecLib.vow(), SB_LERP);
+        // Revoking Content Production Budget (MKT-001)
+        // https://mips.makerdao.com/mips/details/MIP40c3SP49
+        DssVestLike(MCD_VEST_DAI).yank(20);
+
+
+        //Core Unit DAI Budget Transfers
+        // https://mips.makerdao.com/mips/details/MIP40c3SP47
+        DssExecLib.sendPaymentFromSurplusBuffer(SNE_001_WALLET, 229_792);
```

Comment:
Looks like the sheet has this number (LFW just edited row).  Proposal indicates this too:
```suggestion
        DssExecLib.sendPaymentFromSurplusBuffer(SNE_001_WALLET, 257_500);
```

Reply:
I see that change. Will commit this.